### PR TITLE
[PT-BR] Translation of files.md

### PR DESCRIPTION
**Pull request by arielalvesdutra**
- Add tsconfig-referece/copy/pt/options/files.md

### CODE REVIEW

**Comment by arielalvesdutra** [comment]
@khaosdoctor, @alvarocamillont, @danilofuchs e @orta.

File: packages/tsconfig-reference/copy/pt/options/files.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+display: "Files"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/files.md]:
```suggestion
display: "Arquivos"
```

File: packages/tsconfig-reference/copy/pt/options/files.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+oneline: "Inclui uma lista de arquivos, não suporta globs"
+---
+
+Especifica uma lista de arquivos permitidos para incluir no programa. Uma erro ocorre se qualquer um dos arquivos não puder ser encontrado.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/files.md]:
```suggestion
Especifica uma lista de arquivos permitidos para incluir no programa. Um erro ocorre se qualquer um dos arquivos não puder ser encontrado.
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]

**Comment by arielalvesdutra** [comment]
I did the adjustments @danilofuchs. Thank you!

**Review by danilofuchs — APPROVED** [review]
LGTM

**Comment by danilofuchs** [comment]
LGTM

**Comment by orta** [comment]
lgtm

**Comment by orta** [comment]
lgtm

**Comment by orta** [comment]
Heh, this PR is just gonna be a few more LGTMs I think

**Comment by arielalvesdutra** [comment]
If necessary, I can make another PR. Just ask me. Thank you!

**Comment by orta** [comment]
No worries, I was working on the [translator automation](https://github.com/orta/code-owner-self-merge) 

Which now works!

**Pull request merged**